### PR TITLE
add item endpoint

### PR DESCRIPTION
**Pull request by ruanmartinelli**
Adds a simple endpoint to fetch items from the API based on an id.

### CODE REVIEW

File: src/schema.js [hunk 1]
```diff
@@ -1,19 +1,39 @@
...
+    by: { type: GraphQLString },
+    descendants: { type: GraphQLInt },
+    id: { type: GraphQLInt },
+    kids: { type: new GraphQLList(GraphQLInt) },
+    score: { type: GraphQLInt },
+    text: { type: GraphQLString },
+    time: { type: GraphQLInt },
+    title: { type: GraphQLString },
+    type: { type: GraphQLString },
+    url: { type: GraphQLString }
```
Comment (jgcmarins) [inline src/schema.js]: each field should have it's own resolver, so you can query them separately.
see here: https://github.com/jgcmarins/graphql-ufc-api/blob/master/src/schema.js

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.17.1",
```
Comment (jgcmarins) [inline package.json:17]: since node 8, axios isn't needed anymore
you can just use `fetch`: `const response = await fetch(url, options);`

**Review by jgcmarins — CHANGES_REQUESTED** [review]
You are doing great.
Just need to change a few things.

File: src/schema.js (hunk 1 above)
Comment (ruanmartinelli) [reply src/schema.js]: Fixed on d3af8f1! Ty

File: package.json (hunk 2 above)
Comment (ruanmartinelli) [reply package.json:17]: Are you referring to the node-fetch library?

**Review by jgcmarins — APPROVED** [review]
Nice job

File: package.json (hunk 2 above)
Comment (jgcmarins) [reply package.json:17]: https://github.com/matthew-andrews/isomorphic-fetch

Comment (ruanmartinelli) [reply package.json:17]: Guess I merged this PR too soon :sweat_smile: 

Anyway, this is fixed on #4.

**Pull request merged**